App/navigators/AuthenticateStack.tsx

```diff
@@ -49,14 +47,13 @@ const AuthenticateStack: React.FC<Props> = ({ setAuthenticated }) => {
   return (
     <Stack.Navigator screenOptions={{ ...defaultStackOptions, presentation: 'transparentModal', headerShown: false }}>
       {firstLogin ? (
-        <Stack.Group>
-          <Stack.Screen name="Terms & Conditions" component={Terms} />
-          <Stack.Screen name="Create 6-Digit Pin" component={PinCreate} initialParams={{ setAuthenticated }} />
-        </Stack.Group>
+        // <Stack.Group>
```

Comment:
We can just remove the groups if they are single screens now

Reply:
Fixed